pkg/clusteridentity/clusteridentity.go

```diff
@@ -125,10 +125,16 @@ func (a *ClusterIdentityAllocator) Run(stopCh <-chan struct{}) {
 	}
 }
 
+type ClusterIdentity struct {
```

Comment:
out of curiosity, why creating another struct to wrap it?

Reply:
I wanted to limit the changes to the interface in the future if there is a need to add new information to the "cluster identity", such as a user-provided cluster name for example. Do you think it is unlikely?

Reply:
Got it, sounds good to me.